recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "3.17.2" %}
+{% set version = "3.19.1" %}
 
 package:
   name: libprotobuf-suite
   version: {{ version }}
 
 source:
   - url: https://github.com/protocolbuffers/protobuf/archive/v{{ version }}/libprotobuf-v{{ version }}.tar.gz
-    sha256: 36f81e03a0702f8f935fffd5a486dac1c0fc6d4bae1cd02c7a32448ad6e63bcb
+    sha256: 87407cd28e7a9c95d9f61a098a53cf031109d451a7763e7dd1253abf8b4df422
     patches:
       - 0001-remove-Werror-from-test-flags.patch  # [ppc64le or aarch64 or s390x]
-      - 0002-remove-failing-json-parser-test.patch  # [s390x]
+      # - 0002-remove-failing-json-parser-test.patch  # [s390x]
```

Comment:
hmm, those patches might be worth being ported?  Not sure if for our builders memory is big enough for those tests on s390x ...

Reply:
See next comment...